Yosemite/Yosemite/Actions/TaxAction.swift

```diff
@@ -13,4 +13,11 @@ public enum TaxClassAction: Action {
     /// Request the Tax Class found in a specified Product.
     ///
     case requestMissingTaxClasses(for: TaxClassRequestable, onCompletion: (TaxClass?, Error?) -> Void)
+
+    /// Retrieve and synchronizes Tax Rates matching the specified criteria.
+    ///
+    case retrieveTaxRates(siteID: Int64,
+                          pageNumber: Int,
+                          pageSize: Int,
+                          onCompletion: (Result<([TaxRate]), Error>) -> Void)
```

Comment:
Same as above, I believe this is a typo, as the type is declared as a tuple of only 1 value:
```suggestion
                          onCompletion: (Result<[TaxRate], Error>) -> Void)
```

Reply:
Yeah, done in 36aa52386d22ecb32fa8acdc895675c88b2d483e